fontforgeexe/pythonui.c

```diff
@@ -275,13 +281,30 @@ static void InsertSubMenus(PyObject *args,GMenuItem2 **mn, int is_cv) {
     }
 }
 
+int append_mnemonic(PyObject* in, unichar_t* mnemonic) {
+    unichar_t temp[2] = {'\0', '\0'};
+    int idx = 0;
+    for (char* p = PyUnicode_AsUTF8(in); *p != '\0'; p++) {
+	if (*p == '_' && *(p+1) != '\0') {
+	    temp[0] = *(p+1);
+	    u_strcat(mnemonic, temp);
+	    return idx;
+	}
+	idx++;
+    }
+    u_strcat(mnemonic, temp);
```

Comment:
surely this would do nothing?

Reply:
Oh, you're right. I'll have to use the guaranteed non-character, U+FFFF, instead of 0. (I attended the last UTC meeting and contribute to Unicode when I can so I'm sure that'll never become a character, they are exceedingly cautious about backwards compatibility.)